services/login-settings/src/LoginSettingsProvider.tsx

```diff
@@ -0,0 +1,103 @@
+// import i18n from '@dhis2/d2-i18n'
+import { useDataQuery } from '@dhis2/app-service-data'
```

Comment:
I'm assuming it's okay to use useDataQuery, but this could also be refactored to remove the peer dependency here